techniques/system/common/1.0/promises.st

```diff
@@ -482,7 +493,7 @@ bundle agent check_log_system
 
     !windows.rsyslogd.!policy_server::
       "/etc/rsyslog.d/rudder-agent.conf"
-        edit_line => append_if_no_lines("#Rudder log system${const.n}if $syslogfacility-text == 'local6' and $programname startswith 'rudder' then @@${server_info.cfserved}:&SYSLOGPORT&${const.n}if $syslogfacility-text == 'local6' and $programname startswith 'rudder' then ~"),
+        edit_line => append_if_no_lines("#Rudder log system${const.n}if $syslogfacility-text == 'local6' and $programname startswith 'rudder' then ${rsyslog_rule_prefix}${server_info.cfserved}:&SYSLOGPORT&${const.n}if $syslogfacility-text == 'local6' and $programname startswith 'rudder' then ~"),
```

Comment:
this is not ok: we need to remove the udp line and add the tcp line if in tcp (and other way round)
maybe we'd need a setconfigvalue or something like that ?